packages/modeling/src/geometries/poly3/invert.js

```diff
@@ -15,6 +15,7 @@ const invert = (polygon) => {
     // Flip existing plane to save recompute
     inverted.plane = plane.flip(plane.create(), polygon.plane)
   }
+  inverted.color = polygon.color;
```

Comment:
This should be conditional, so you dont attach undefined on all polygons

Reply:
Thank you, and agreed. I've just added a commit.